modules/ding_library/ding_library.features.field_instance.inc

```diff
@@ -705,6 +705,64 @@ function ding_library_field_default_field_instances() {
     ),
   );
 
+  // Exported field_instance: 'node-ding_library-field_ding_library_type'.
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-937

Reply:
This case does not describe this change.

Reply:
Think this was added to aid theming?
The result:
![image](https://user-images.githubusercontent.com/574498/98229562-bd698f80-1f62-11eb-9c3c-1a12fb9ae891.png)

Reply:
Oh, so its a completely new field on the Library CT?

Reply:
I think it's a good feature to have a "Type" definition of a Library CT. It's important that we have features that will generate the vocab and define default terms.

Reply:
The taxonomy vocabulary feature is there, however the terms are not and should be custom made.

Reply:
As long as the field is not mandatory, it's OK that there is no default terms added.